src/linker/Linker/Driver.cs

```diff
@@ -266,6 +267,12 @@ protected int SetupContext (ILogger customLogger = null)
 
 						continue;
 
+					case "--mark-assembly-step":
```

Comment:
I think if we stay with an interface that is very close to the existing custom step it would be better to use the same command line option

Reply:
I tend to slightly prefer just reusing `--custom-step` and relying on which interface the class implements.